utils/hash/compare.js

```diff
@@ -10,4 +8,4 @@
  *
  * @example hash.compare('a1b22c23d24e25f4g33a123b23c34', 'a1b22c23d24e25f4g33a123b23c34');
  */
-module.exports = (hash1, hash2) => hash1 === hash2;
+export default (hash1, hash2) => hash1 === hash2;
```

Comment:
A bit of an aside, but does this need to be its own function? 😅

Reply:
Nah but I'm not touching the structure in this PR. Thats a black hole to be avoided when ESMing 😆